isis/tests/TestCsmModel.cpp

```diff
@@ -4,8 +4,35 @@
 #include <nlohmann/json.hpp>
 using json = nlohmann::json;
 
+const std::string TestCsmModel::SENSOR_MODEL_NAME = "TestCsmModelName";
+const std::vector<std::string> TestCsmModel::PARAM_NAMES = {
+  "TestParam1",
```

Comment:
Update this or Plugin and example ISD to match? I had TestCsmModel only have TestParam1 and TestParam2